### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -11,14 +11,21 @@ export class App extends React.Component<{}, State> {
   };
 
   addOne = () => {
-    // write code here
+    this.setState((state) => {
+      return {
```
Comment (OlhaYevstifieieva) [inline src/App.tsx:15]: You don`t need in a redundant 'return'

**Review by OlhaYevstifieieva — APPROVED** [review]
GJ!

File: src/App.tsx (hunk 1 above)
Comment (ArturErmolenko) [reply src/App.tsx:15]: agree

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
Screenshot 2023-01-31 at 19 02 18
😎

**Review by ArturErmolenko — APPROVED** [review]
Nice

**Pull request closed without merging**